wordpress-mobile/WordPress-Android
# Add tracking for enabling/disabling of system notifications

Fixes #10581
This PR adds a check on the app start which checks whether the system notifications are enabled or disabled. When the notification status changes, we track either `notifications_enabled` or `notifications_disabled` event.
I've also added a `system_notifications_enabled ` flag to `NOTIFICATION_RECEIVED_PROCESSING_START` and to `NOTIFICATION_RECEIVED_PROCESSING_END` which should helps us recognise which notifications are ignored because they are not shown.  

To test:
* Disable notifications for the WordPress app
* Go to the app
* Notice in the LogCat that `notifications_disabled` is tracked
* Restart the app
* Notice that `notifications_disabled` is not tracked again
* Enable notifications for the WordPress app
* Notice in the LogCat that `notifications_enabled` is tracked
* Restart the app
* Notice that `notifications_enabled` is not tracked again

PR submission checklist:
